torch/csrc/jit/argument_spec.h

```diff
@@ -153,7 +153,8 @@ struct ArgumentSpec {
 
  private:
   TypePtr fillType(TypePtr original, size_t& offset) const {
-    if (original->isSubtypeOf(TensorType::get())) {
+    if (original->isSubtypeOf(TensorType::get())
+	|| original->isSubtypeOf(OptionalType::ofTensor())) {
```

Comment:
This is quite surprising. Do we seriously translate `None` to an undefined tensor when you call a script function? How can that even work?

Reply:
Oh, is it because all args are zero-initialized, so you're abusing the fact that this arg is not a tensor, but it will report being undefined? Bleh